### PR TITLE
replace unneeded DI container parameters

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: Resources/config/body_listener.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <service id="fos_rest.decoder.xml" class="FOS\RestBundle\Decoder\XmlDecoder" />
 
         <service id="fos_rest.decoder_provider" class="FOS\RestBundle\Decoder\ContainerDecoderProvider">
-            <argument>%fos_rest.decoders%</argument>
+            <argument /> <!-- decoders -->
```
Comment (stof) [inline Resources/config/body_listener.xml]: I would use ``<argument type="collection">`` here, so that the non-replaced argument is already an array

Comment (lsmith77) [inline Resources/config/body_listener.xml]: +1

Comment (xabbuh) [inline Resources/config/body_listener.xml]: good idea

**Comment by lsmith77** [comment]
There is an error now:
https://travis-ci.org/FriendsOfSymfony/FOSRestBundle/jobs/84038805#L301

This is caused by https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1156 which now uses the ``fos_rest.mime_types`` in two more places.

**Comment by xabbuh** [comment]
I rebased and fixed the two parts.

File: Resources/config/request_body_param_converter.xml [hunk 2]
```diff
@@ -8,10 +8,10 @@
 
         <service id="fos_rest.converter.request_body" class="FOS\RestBundle\Request\RequestBodyParamConverter">
             <argument type="service" id="fos_rest.serializer"/>
-            <argument>%fos_rest.serializer.exclusion_strategy.groups%</argument>
-            <argument>%fos_rest.serializer.exclusion_strategy.version%</argument>
+            <argument /> <!-- serializer exclusion strategy groups -->
```
Comment (stof) [inline Resources/config/request_body_param_converter.xml]: shouldn't this be a collection rather than an empty string when loading the file ?

Comment (xabbuh) [inline Resources/config/request_body_param_converter.xml]: You are right. I also found some more places where such a change was needed.

**Pull request merged**